ipalib/constants.py

```diff
@@ -279,6 +279,8 @@
 DOMAIN_SUFFIX_NAME = 'domain'
 CA_SUFFIX_NAME = 'ca'
 PKI_GSSAPI_SERVICE_NAME = 'dogtag'
+PKI_CA_SIGNING_KEY_SIZE = '4096'
```

Comment:
4096 is way too large as default and goes way beyond all recommendations. CA and OCSP should be either 2048 or 3072, EE certs 2048bit.

Reply:
We can change this when the actual agreement will be in place. For now, it is enough to demonstrate that certain code paths are failing because of the key size not being 2048. I'd rather concentrate on fixing those ones rather than discussing the defaults. Note that I specifically made a default-changing commit a separate one to not overlap with the real effort that needs to be done.

Reply:
For a long term CA key, 4096 is correct at this point, given the rapid progress in QC.
2048 is considered ok for shorter lived certificates though.

Reply:
3072 is good enough for CA keys. 4096 is really slow, especially on low powered IoT devices.